scripts/collect-info.yaml

```diff
@@ -7,11 +7,9 @@
     debug: false
     commands:
       - name: git_info
-        cmd: cat {kargo,.}/.git/logs/HEAD
+        cmd: cat .git/logs/HEAD
```

Comment:
changed: [node1] => (item={u'cmd': u'cat .git/logs/HEAD', u'name': u'git_info'}) => {"changed": true, "cmd": "cat .git/logs/HEAD 2>&1 | tee git_info", "delta": "0:00:00.017041", "end": "2016-08-22 16:41:12.059758", "item": {"cmd": "cat .git/logs/HEAD", "name": "git_info"}, "rc": 0, "start": "2016-08-22 16:41:12.042717", "stderr": "", "stdout": "cat: .git/logs/HEAD: No such file or directory", "stdout_lines": ["cat: .git/logs/HEAD: No such file or directory"], "warnings": []}

By default kargo is checked out to ~/kargo, so why not use this relative directory? We could set a var instead.